src/utils/version-util.ts

```diff
@@ -0,0 +1,37 @@
+import * as semver from 'semver';
+import { isNumber } from 'util';
+
+function isWindowsComplaint(version: string): boolean {
```

Comment:
`isWindowsComplaint` would probably always return `true`, you may have wanted `isWindowsCompliant` :P

Reply:
Haha yeah we could complain about it to. :) 
This will return false because you can now pass in semantic version strings.